Tests errors

I got three errors when running python setup.py test. This happens with freshly cloned pyfolio from master.

First one seems to be a missing definition

```
-------------------- >> begin captured logging << --------------------
root: DEBUG: Found kernel cling in /usr/local/share/jupyter/kernels
root: DEBUG: Native kernel (python2) available from /Users/javier/anaconda/lib/python2.7/site-packages/ipykernel/resources
root: DEBUG: Connecting to: tcp://127.0.0.1:58295
root: DEBUG: connecting shell channel to tcp://127.0.0.1:58292
root: DEBUG: Connecting to: tcp://127.0.0.1:58292
root: DEBUG: connecting iopub channel to tcp://127.0.0.1:58293
root: DEBUG: Connecting to: tcp://127.0.0.1:58293
root: DEBUG: connecting stdin channel to tcp://127.0.0.1:58294
root: DEBUG: Connecting to: tcp://127.0.0.1:58294
root: DEBUG: connecting heartbeat channel to tcp://127.0.0.1:58296
root: INFO: Running cell:
%matplotlib inline
import pyfolio as pf

root: INFO: Cell returned
root: INFO: Running cell:
stock_rets = pf.utils.get_symbol_rets('FB')

root: INFO: Cell returned
root: INFO: Running cell:
out_of_sample = stock_rets.index[-40]

root: INFO: Cell returned
root: INFO: Running cell:
pf.create_bayesian_tear_sheet(stock_rets, live_start_date=out_of_sample)

root: INFO: Cell raised uncaught exception: 
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-4-fcbdb054fd92> in <module>()
----> 1 pf.create_bayesian_tear_sheet(stock_rets, live_start_date=out_of_sample)

/Users/javier/code/github/pyfolio/pyfolio/plotting.pyc in call_w_context(*args, **kwargs)
     42         if set_context:
     43             with context():
---> 44                 return func(*args, **kwargs)
     45         else:
     46             return func(*args, **kwargs)

/Users/javier/code/github/pyfolio/pyfolio/tears.pyc in create_bayesian_tear_sheet(returns, benchmark_rets, live_start_date, samples, return_fig)
    525 
    526     # Run T model with missing data
--> 527     trace_t = bayesian.run_model('t', df_train, returns_test=df_test,
    528                                  samples=samples)
    529 

NameError: global name 'bayesian' is not defined
--------------------- >> end captured logging << ---------------------
```

The next two errors are numerical values:

```
======================================================================
FAIL: test_sortino_0 (pyfolio.tests.test_timeseries.TestStats)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/javier/code/github/pyfolio/.eggs/nose_parameterized-0.5.0-py2.7.egg/nose_parameterized/parameterized.py", line 365, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/Users/javier/code/github/pyfolio/pyfolio/tests/test_timeseries.py", line 297, in test_sortino
    expected)
AssertionError: -35490980255632276.0 != -458003439.10738045

======================================================================
FAIL: test_sortino_1 (pyfolio.tests.test_timeseries.TestStats)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/javier/code/github/pyfolio/.eggs/nose_parameterized-0.5.0-py2.7.egg/nose_parameterized/parameterized.py", line 365, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "/Users/javier/code/github/pyfolio/pyfolio/tests/test_timeseries.py", line 297, in test_sortino
    expected)
AssertionError: -56038389877314144.0 != -723163324.9063905
----------------------------------------------------------------------
```

--- a/pyfolio/tests/test_nbs.py
+++ b/pyfolio/tests/test_nbs.py
@@ -16,6 +16,14 @@ from pyfolio.utils import pyfolio_root
 def test_nbs():
     path = os.path.join(pyfolio_root(), 'examples', '*.ipynb')
     for ipynb in glob.glob(path):
+
+        # See if bayesian is useable before we run a test
+        if ipynb.endswith('bayesian.ipynb'):
+            try:
+                import pyfolio.bayesian  # NOQA
+            except:
+                continue
+
         with open(ipynb) as f:
             nb = read(f, 'json')
             nb_runner = NotebookRunner(nb)

--- a/pyfolio/tests/test_nbs.py
+++ b/pyfolio/tests/test_nbs.py
@@ -20,7 +20,7 @@ def test_nbs():
         # See if bayesian is useable before we run a test
         if ipynb.endswith('bayesian.ipynb'):
             try:
-                import pyfolio.bayesian  # NOQA
+                import pymc3  # NOQA
             except:
                 continue
 

